model/user/user.go

```diff
@@ -21,25 +21,25 @@ import (
 )
 
 type DBUser struct {
-	Id                     string           `bson:"_id"`
-	FirstName              string           `bson:"first_name"`
-	LastName               string           `bson:"last_name"`
-	DispName               string           `bson:"display_name"`
-	EmailAddress           string           `bson:"email"`
-	PatchNumber            int              `bson:"patch_number"`
-	PubKeys                []PubKey         `bson:"public_keys" json:"public_keys"`
-	CreatedAt              time.Time        `bson:"created_at"`
-	Settings               UserSettings     `bson:"settings"`
-	APIKey                 string           `bson:"apikey"`
-	SystemRoles            []string         `bson:"roles"`
-	LoginCache             LoginCache       `bson:"login_cache,omitempty"`
-	FavoriteProjects       []string         `bson:"favorite_projects"`
-	OnlyAPI                bool             `bson:"only_api,omitempty"`
-	ParsleyFilters         []parsley.Filter `bson:"parsley_filters"`
-	ParsleySettings        parsley.Settings `bson:"parsley_settings"`
-	NumScheduledPatchTasks int              `bson:"num_scheduled_patch_tasks"`
-	LastScheduledTasksAt   time.Time        `bson:"last_scheduled_tasks_at"`
-	BetaFeatures           BetaFeatures     `bson:"beta_features"`
+	Id                     string                 `bson:"_id"`
+	FirstName              string                 `bson:"first_name"`
+	LastName               string                 `bson:"last_name"`
+	DispName               string                 `bson:"display_name"`
+	EmailAddress           string                 `bson:"email"`
+	PatchNumber            int                    `bson:"patch_number"`
+	PubKeys                []PubKey               `bson:"public_keys" json:"public_keys"`
+	CreatedAt              time.Time              `bson:"created_at"`
+	Settings               UserSettings           `bson:"settings"`
+	APIKey                 string                 `bson:"apikey"`
+	SystemRoles            []string               `bson:"roles"`
+	LoginCache             LoginCache             `bson:"login_cache,omitempty"`
+	FavoriteProjects       []string               `bson:"favorite_projects"`
+	OnlyAPI                bool                   `bson:"only_api,omitempty"`
+	ParsleyFilters         []parsley.Filter       `bson:"parsley_filters"`
+	ParsleySettings        parsley.Settings       `bson:"parsley_settings"`
+	NumScheduledPatchTasks int                    `bson:"num_scheduled_patch_tasks"`
+	LastScheduledTasksAt   time.Time              `bson:"last_scheduled_tasks_at"`
+	BetaFeatures           evergreen.BetaFeatures `bson:"beta_features"`
```

Comment:
I think it would make sense for this to be in the same place as UseSpruceOptions. It may be too late to move it without it being annoying at this point but if it's still easy I'd advocate for that. I'll leave it up to you though.